packages/dom-expressions/src/server.js

```diff
@@ -4,6 +4,8 @@ import stringify from "./serializer";
 export { stringify };
 export { createComponent } from "rxcore";
 
+// Based on https://github.com/WebReflection/domtagger/blob/master/esm/sanitizer.js
+const VOID_ELEMENTS = /^(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr)$/i;
```

Comment:
This was carried over from [`lit-dom-expressions`](https://github.com/ryansolid/dom-expressions/blob/main/packages/lit-dom-expressions/src/index.ts#L35)